src/app/app.routes.ts

```diff
@@ -0,0 +1,47 @@
+import { provideRouter, RouterConfig } from '@angular/router';
+import { MapViewComponent } from './views/map-view/map-view.component';
+import { CatViewComponent } from './views/cat-view/cat-view.component';
+import { CatHelpComponent } from './widgets/cat-help/cat-help.component';
+import { CatSource2FHLComponent } from './widgets/cat-source/cat-source-2fhl/cat-source-2fhl.component';
+import { CatSource3FGLComponent } from './widgets/cat-source/cat-source-3fgl/cat-source-3fgl.component';
+import { CatSourceSNRcatComponent } from './widgets/cat-source/cat-source-snrcat/cat-source-snrcat.component';
```

Comment:
Could try this here in `src/app/app.routes.ts` (or from anywhere else):
```
import { CatSourceSNRcatComponent } from './widgets/cat-source';
```
And in `app/widgets/cat-source/index.ts` put this:
```
export { CatSourceSNRcatComponent } from './cat-source-snrcat/cat-source-snrcat.component'
```